config/locales/models/organisation.fr.yml

```diff
@@ -2,3 +2,10 @@ fr:
   activerecord:
     models:
       organisation: Organisation
+    attributes:
+      organisation:
+        name: Nom
+        email: Email
+        phone_number: Numéro de téléphone
+        slug: Désignation pour fichier allocataires
```

Comment:
```suggestion
        slug: Désignation dans le fichier allocataires
``` 
WDYT?

Reply:
sure, why not. J'ai eu du mal à trouver quelque chose ici, j'ai presqu'hésité à laisser "slug"